watsononlinestore/watson_online_store.py

```diff
@@ -54,7 +58,10 @@ def __init__(self, email=None, first_name=None, last_name=None,
         self.shopping_cart = shopping_cart
 
     def get_customer_dict(self):
-        """ Specific to our cloudant_online_store
+        """ Returns a dict in form usable by our cloudant_online_store DB
```

Comment:
I'm pretty sure no space after """ although I never liked that look. Maybe this is allowed(?)